doc/sphinx/source/releasenotes.rst

```diff
@@ -7,7 +7,7 @@ for each release of libCEED.
 
 .. _v0.6:
 
-v0.6 (Mar 29, 2020)
+v0.6 (Mar 30, 2020)
```

Comment:
It was Mar 29 on the west coast. ;-)

Reply:
Ahah I know... but GitHub is not forgiving :sweat_smile: I thought we should be consistent with it
![Screenshot from 2020-04-01 09-17-15](https://user-images.githubusercontent.com/39932030/78154763-de309c00-73f9-11ea-9bf3-d89241185582.png)

Reply:
That's the client's timezone, FWIW. Releases pretty much always look a day off in Japan, for example; it's impossible to be globally consistent.
![Screenshot_20200401_093315](https://user-images.githubusercontent.com/3303/78162590-f0abd500-73fb-11ea-99a2-ff026c8bedd9.png)

Reply:
Ah, I wasn't sure if it  was the client's one or not. Ok then, I can put it back